templates/forms/default/form.html.twig

```diff
@@ -49,6 +53,9 @@
       {% endblock %}
       {% if form.novalidate %}novalidate{% endif %}
       {% if form.keep_alive %}data-grav-keepalive="true"{% endif %}
+      {% for k, v in form.data.blueprints.form %}
```

Comment:
I think it would be cool to make this even better to have something like this
```suggestion
      {% for attribute, value in form.data.blueprints.form.attributes %}
          { attribute }}="{{ value|e }}"
      {% endfor %}
```
It would allow you to pass any attribute to the form instead of just inserting data-*

Reply:
I'm not against it (there may be legitimate uses?) but I don't have a use-case for it myself and the data nesting structure is quite complex. Maybe to postpone as later enhancement from someone else?

BTW:
* `form.data.blueprints.form.attributes` does not exists
* adding non `data-*` likely means providing non-standard attributes to resulting HTML

Reply:
ping?